modules/tinymce/src/core/test/ts/browser/content/insert/NamespaceElementsTest.ts

```diff
@@ -11,12 +11,13 @@ describe('browser.tinymce.core.content.insert.NamespaceElementsTest', () => {
     valid_children: '+p[math]'
   }, [], true);
 
-  it('TINY-10237: Inserting SVG elements but filter out things like scripts', () => {
+  // Updated for TINY-11332: Remove html inside SVG
```

Comment:
I wonder if we need these comments and have a separate test that just validates this specific new behaviour and update the old tests to not have anything but scripts and text in the input data?

Reply:
Removed comments and added new test